doc/api/documentation.md

```diff
@@ -32,6 +32,17 @@ The stability indices are as follows:
 > [semantic versioning][] rules. Non-backward compatible changes or removal may
 > occur in any future release. Use of the feature is not recommended in
 > production environments.
+>
+> Experimental features are subdivided into stages:
+>
+> * 1.0 - Early development. Experimental features at this stage are unfinished
```

Comment:
It looks like the descriptions of these subdivisions conflict in meaning with what has already been presented above  - https://github.com/nodejs/node/blob/0593b699b8a4cd517058ef68576a460e01451a18/doc/api/documentation.md?plain=1#L31-L34 because that suggests that breaking changes / removal is expected at every stage but by reading the subdivisions I get the feeling that a user shouldn't expect those in stage 1.2.

Reply:
Updated.